backend/auth/handlers/auth_test.go

```
@@ -0,0 +1,186 @@
+		},
+	}
+	refreshRepo := &testutil.FakeRefreshTokenRepository{
+		InsertFunc: func(ctx context.Context, token *domains.RefreshToken) *serviceresponse.Response[any] {
+			return nil
+		},
+	}
+
+	h := newTestAuthHandler(t, gateway, authRepo, refreshRepo)
+
+	body, _ := json.Marshal(map[string]string{"email": "user@example.com", "password": testPassword})
+	req := httptest.NewRequest(http.MethodPost, "/v1/auth/login", bytes.NewReader(body))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)